### DISCUSSION TITLE
BUG: opgevoerde naam wijzigt wanneer je op bewerken klikt:

### DISCUSSION THREAD

**Discussion by CelineJansen** [kadaster-labs/sensrnet-home#304] [Q&A]
Beacon via API opgevoerd met naam "Citybeacon EHV-13":
![image](https://user-images.githubusercontent.com/79267472/160775908-da3714c5-0f6f-410f-a7d5-17d85f86e905.png)

In het menu "uitgebreid bewerken" maakt de tool automatisch "Baken" van de naam:
![image](https://user-images.githubusercontent.com/79267472/160776103-b3c03c8f-de23-4b3c-a6f7-0e773f7a3e07.png)

Die kan ik vervolgens niet eens opslaan, want "Baken" is minder dan 6 tekens (gelukkig maar in dit geval ;), anders was me dit niet opgevallen...)

**Comment by marcvanandel** [comment]
@kad-floriw , hadden we deze bug niet al opgelost? (en is de versie van Eindhoven verouderd?)

**Reply by kad-floriw** [reply]
Dat klopt inderdaad, die bug is laatst opgelost door stefan. Had je die versie ook al uitgerold @kad-busses?
